src/ui/sass/modules/_SearchBar.scss

```diff
@@ -194,6 +194,7 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
 
   &-LoadingIndicator
   {
+    display: block;
```

Comment:
I don't think we need to add this style on this class because it's only used for elements that already have `yxt-SearchBar-AnimatedIcon` which has display: flex